src/test/java/com/thomashan/coup/PlayerBuilder.java

```diff
@@ -9,7 +9,17 @@ public static Player build(boolean active) {
             return player;
         }
 
-        return player.revealCard1().revealCard2();
+        return revealAllCards(player);
+    }
+
+    private static Player revealAllCards(Player playerToRevealCards) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found 'UR'-anomaly for variable 'playerCard' (lines '15'-'19').](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)